_component_design/dropdowns.md

```diff
@@ -15,20 +15,20 @@ need: selectors
 
 {% include todo.html content="Update style and add additional examples" %}
 
-Use the simple selector when choosing from a limited number options. If more
-than 10-12 options need to displayed, use the complex selector, which contains
-a search. Always use the complex selector for devices. Be mindful of the length
+<!-- Use the simple selector when choosing from a limited number of options. Use the
```

Comment:
hi. please explain why this is commented out?

Reply:
Please see above :)

Reply:
kk thanks